samples/dash-if-reference-player/app/main.js

```diff
@@ -1815,6 +1830,12 @@ app.controller('DashController', ['$scope', '$window', 'sources', 'contributors'
                 case 'drmToday':
                     $scope.drmToday = this.parseBoolean(value);
                     break;
+                case 'enableRollUp':
```

Comment:
We should not need these fields here as these two attributes are defined in `Settings.js`.